packages/storybook/.storybook/preview.tsx

```diff
@@ -38,3 +37,11 @@ export const parameters = {
     page: null,
   },
 };
+
+export const decorators = [
+  (Story) => (
+    <React.StrictMode>
```

Comment:
So not using https://github.com/storybookjs/storybook/issues/12734 😆

Reply:
Nope! I tried but it didn't seem to work